docs/understanding-airbyte/operations.md

```diff
@@ -0,0 +1,51 @@
+# Sync Operations
+
+Airbyte [connections](connections/README.md) support configuring additional transformations that execute after the sync. Useful applications could be:
+
+- Customized normalization to better fit the requirements of your own business context.
+- Business transformations from a technical data representation into a more logical and business oriented data structure. This can facilitate usage by end-users, non-technical, executives for Business Intelligence dashboard and report.
+- Data Quality, performance optimization, alerting and monitoring etc
+- Integration with other tools from your data stack (orchestration, data visualization, etc)
+
+## Supported Operations
+
+### dbt transformations
+
+#### - git repository url:
+A url to a git repository to (shallow) clone the latest dbt project code from.
+
+The project versioned in the repository is expected to:
+
+- be a valid dbt package with a `dbt_project.yml` file at its root.
+- the `dbt_project.yml` file should declare some "profile" name as described [here](https://docs.getdbt.com/dbt-cli/configure-your-profile).
+
+When using dbt cli, dbt checks your `profiles.yml` file for a profile with the same name. A profile contains all the details required to connect to your data warehouse. This file generally lives outside of your dbt project to avoid sensitive credentials being check in to version control. Therefore, a `profiles.yml` will be generated according to the configured destination from Airbyte UI.
```

Comment:
```suggestion
When using the dbt CLI, dbt checks your `profiles.yml` file for a profile with the same name. A profile contains all the details required to connect to your data warehouse. This file generally lives outside of your dbt project to avoid sensitive credentials being checked in to version control. Therefore, a `profiles.yml` will be generated according to the configured destination from the Airbyte UI.
```